### PR TITLE
Add v8

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: config/implementation/v8.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "license": "V8",
+  "targetGit": "https://github.com/tc39/test262.git",
+  "targetSubDirectory": "test262/implementation-contributed/v8",
+  "curationLogsPath": "test262/implementation-contributed/curation_logs/v8.json",
+  "implementerName": "V8",
+  "implementerDisplayName": "V8",
```
Comment (nomadtechie) [inline config/implementation/v8.json]: lowercase `V8` for `implementerName` - the `implementerDisplayName` looks fine

**Comment by nomadtechie** [comment]
It would be nice to have a `debug` version of this which mock repos for testing...can you file an issue to track this @rwaldron ?

File: config/implementation/v8.json (hunk 1 above)
Comment (rwaldron) [reply config/implementation/v8.json]: All set!

**Comment by rwaldron** [comment]
- Comment addressed
- Fixups squashed
- Filed: https://github.com/bocoup/test262-automation/issues/80

**Pull request merged**